tests/test_tydiqa_primary.py

```diff
@@ -0,0 +1,19 @@
+from datasets import load_dataset
+from transformers import AutoTokenizer
+
+from evaluation.tasks.tydiqa_primary.tydiqa_primary import TyDiQADataset
+
+
+def test_prompt():
+    data = load_dataset("tydiqa", "secondary_task", split="validation")
+    tokenizer = AutoTokenizer.from_pretrained("gpt2")
+    tokenizer.pad_token = tokenizer.eos_token
+    tokenizer.padding_side = "left"
+    dataset = TyDiQADataset(data, tokenizer, ["english"])
+    prompt = dataset[0]["prompt"]
+    assert (
+        "Wound care encourages and speeds wound healing via cleaning and protection from reinjury or infection. "
+        "Depending on each patient's needs, it can range from the simplest first aid to entire nursing specialties "
+        "such as wound, ostomy, and continence nursing and burn center care.\n"
+    ) in prompt
```

Comment:
This is just a demo, not really a good test case. For better test cases, perhaps more PRs wanted.